composer.json

```diff
@@ -14,7 +14,8 @@
     },
     "autoload": {
         "psr-4": {
-            "DivideBV\\PHPDivideIQ\\": ["src"]
+            "DivideBV\\PHPDivideIQ\\": ["src"],
+            "DivideBV\\PHPDivideIQ\\Models\\": ["src/Models"]
```

Comment:
Thought this was needed, but its not. Fixed it in commit: https://github.com/Nav-Appaiya/PHPDivideIQ/commit/816664780ea9207199a47c1116eee367c845eb33